src/plugin.rs

```diff
@@ -242,20 +293,46 @@ impl<R: Runtime> Plugin<R> for TauriSql<R> {
             } else {
                 serde_json::from_value(config)?
             };
+
+            #[cfg(feature = "sqlite")]
+            let app_path = get_app_path(app);
+
+            #[cfg(feature = "sqlite")]
+            create_dir_all(&app_path).expect("problems creating App directory");
+
+            #[cfg(feature = "sqlite")]
+            let path_mapper = move |p: String| -> String {
+                format!(
+                    "sqlite:{}",
+                    Path::new(&app_path.as_str())
+                        .join(p)
+                        .to_str()
+                        .unwrap()
+                        .to_string()
+                )
+            };
+
+            #[cfg(not(feature = "sqlite"))]
+            let path_mapper = |p: String| p;
```

Comment:
`PathBuf` doesn't have a `to_str()` method, right? Only Path does.

Reply:
@ksnyde In theory that's true, but PathBuf can use all of Path's methods without manual converting it, because it [implements](https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.to_str) `Deref<Path>`. Otherwise you current method wouldn't work either, because `.join()` [returns a PathBuf](https://doc.rust-lang.org/std/path/struct.Path.html#method.join)!